packages/debug/package.json

```diff
@@ -38,8 +38,9 @@
   ],
   "dependencies": {
     "@types/debug": "4.1.7",
+    "debug": "4.3.4",
```

Comment:
Note, here we use the `debug` npm package now and not the harcoded version of it (which was deleted in this PR)

Reply:
nice!